report/pom.xml

```diff
@@ -27,10 +27,15 @@
             <artifactId>gson</artifactId>
             <version>2.8.2</version>
         </dependency>
+        <dependency>
+            <groupId>org.egov</groupId>
+            <artifactId>enc-client</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
         <dependency>
             <groupId>org.egov.services</groupId>
             <artifactId>services-common</artifactId>
-            <version>0.5.0</version>
+            <version>1.0.1-SNAPSHOT</version>
```

Comment:
Why is this dependency updated?

Reply:
There was need of uuid field in "org.egov.common.contract.request.User" class, which was absent in "0.5.0" version.